pootle/apps/pootle_store/models.py

```diff
@@ -873,6 +873,7 @@ def resurrect(self, is_fuzzy=False):
             self.state = UNTRANSLATED
 
         self.update_qualitychecks(keep_false_positives=True)
+        self.index = self.store.max_index() + 1
```

Comment:
im not sure if it works correctly - i need to test further - but i think the indeces get updated after resurrection. If not we might need to change the order in the update